dune/xt/functions/constant.hh

```diff
@@ -92,6 +92,46 @@ public:
 };
 
 
+template <class Element, size_t rangeDim = 1, size_t rangeDimCols = 1, class RangeField = double>
+class ConstantGridFunction : public GridFunctionInterface<Element, rangeDim, rangeDimCols, RangeField>
+{
+  using BaseType = GridFunctionInterface<Element, rangeDim, rangeDimCols, RangeField>;
+
+public:
+  using typename BaseType::LocalFunctionType;
+
+  explicit ConstantGridFunction(const typename LocalFunctionType::RangeType& constant,
+                                const std::string name_in = static_id())
+    : constant_function_(constant, name_in)
+    , constant_grid_function_(constant_function_)
+  {}
+
+  explicit ConstantGridFunction(const RangeField& constant, const std::string name_in = static_id())
```

Comment:
This ctor is problematic in 1d, I would just drop it if you don't require it.

Reply:
I just copied the constructors from ``ConstantFunction``, if it works there it should work here, I hope. I will add a test.